packages/components/select/src/select.js

```diff
@@ -20,6 +20,12 @@ export class TSSelect extends TSElement {
 		this._translations = Object.assign({}, translations);
 		this.handleInputDebounced = helpers.debounceEvent(() => {
 			this.filterValue = this.inputValue;
+			/**
+			 * Emitted when filter value of the select changes. You can listen to this to update the
```

Comment:
I think you lost end of the sentence. Please, add the information, what they can update from this event.